### DISCUSSION TITLE
where put the .password

### DISCUSSION THREAD

**Discussion by Rigolino1** [spencerwooo/onedrive-vercel-index#856] [Q&A]
were put my password, the tuto say 'on the route of you want protect', i mean put the .password file on my onedrive protect route?

**Comment by spencerwooo** [comment]
Suppose the directory to protect is `/foo/bar/protect` on your OneDrive, you should then simply put `.password` under this directory, where the final path of `.password` should be:

```
/foo/bar/protect/.password
```
